CHANGELOG.md

```diff
@@ -18,6 +18,12 @@
 * [Breaking] Rename `yarpc.CreateThriftServiceFunc` to `yarpc.ServiceCreateFunc`
   as it is not thrift-specific.
 * Report version metrics for company-wide version usage information.
+* [Breaking] Rename functions inside `package dig` to match other APIs within UberFx:
+        * `Register` to `TryRegister`
```

Comment:
Also have a preference for dropping the Try*.
Methods that return an error -> Register, Resolve
Methods than panic -> MustRegister, MustResolve

Just sounds more natural. Maybe I've seen more usages of Must than Try.